docs/modules/ROOT/pages/configure-cmos.adoc

```diff
@@ -69,6 +84,10 @@ PROMETHEUS_DYNAMIC_INTERNAL_DIR=${PROMETHEUS_DYNAMIC_INTERNAL_DIR:-/etc/promethe
 
 === Endpoints
 
+[NOTE]
+This is a legacy way of configuring Prometheus targets and we recommend not to use this approach.
+You can use CMOS UI to perform the same using the "Add Cluster" option present on the landing page.
```

Comment:
I know you just moved it (sorry!) - would it be best to put it at the top of the _above_ section (i.e. above the manual cluster monitor REST API, as that's what the form does under the hood anyway)?

Reply:
sure